### PR TITLE
[Scheduler] Enrich kind label

### PR DESCRIPTION
**Pull request by roei3000b**
If kind label doesn't exists, set it.

https://iguazio.atlassian.net/browse/ML-6695

### CODE REVIEW

**Review by moranbental — CHANGES_REQUESTED** [review]
Maybe create unit tests to check that the kind is really enriched

File: server/api/utils/scheduler.py [hunk 1]
```diff
@@ -347,9 +349,9 @@ def store_schedule(
 
         self._ensure_auth_info_has_access_key(auth_info, kind)
         secret_name = self._store_schedule_secrets_using_auth_secret(auth_info)
-        labels = self._append_access_key_secret_to_labels(labels, secret_name)
+        labels = self._enrich_labels(labels, secret_name)
+        labels.setdefault(mlrun.common.constants.MLRunInternalLabels.kind, kind)
```
Comment (rokatyy) [inline server/api/utils/scheduler.py]: why not like this? Do we need explicit replacement in that case?

```
labels = self._enrich_labels(labels, secret_name, kind)
```

Comment (roei3000b) [reply server/api/utils/scheduler.py]: OOPS, I forgot to delete this line.

**Review by rokatyy — COMMENTED** [review]
I've found this test https://github.com/mlrun/mlrun/blob/109ca7d115b8790c165726fc1136e241e42774c7/tests/api/utils/test_scheduler.py#L1466, maybe just worth checking that it covers this part and we're done :)

**Review by alonmr — CHANGES_REQUESTED** [review]
~You are confusing schedule kinds with runtime kinds 😅 . This is not the correct label value.~ - scratch that

**Review by alonmr — COMMENTED** [review]
I'm not sure I'm following the fix. Please explain the issue and how this solves it. 
When the scheduler triggers the job, you want it to have the runtime kind label value not the schedule kind. So what is this for? I don't think you should use the labels to determine the schedule kind if you have it in the schedule object

**Comment by roei3000b** [comment]
> I'm not sure I'm following the fix. Please explain the issue and how this solves it. When the scheduler triggers the job, you want it to have the runtime kind label value not the schedule kind. So what is this for? I don't think you should use the labels to determine the schedule kind if you have it in the schedule object

Oh, you right, this pr is wrong.
I will open another one.

**Pull request closed without merging**